.vscodeignore

```diff
@@ -11,6 +11,7 @@ examples/**
 images/*.psd
 images/*.ai
 images/screenshot_folders.png
+images/screenshot890x486.gif
```

Comment:
Since you are using internal path in README.md to the image you need to remove this line so the image gets into the distro.